### PR TITLE
home work 1

### PR DESCRIPTION
**Pull request by Julia-92**
Home Work 1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,104 +1,3 @@
...
-# FuseBox cache
-.fusebox/
-
-# DynamoDB Local files
-.dynamodb/
-
-# TernJS port file
-.tern-port
+node_modules
+packege-lock.json
```
Comment (adudyak) [inline .gitignore:2]: typo issue in name

File: pages/created_account.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const { I } = inject();
+
+module.exports = {
+
+  verifyCreatedAccountPageName() {
```
Comment (adudyak) [inline pages/created_account.js]: Move this method to register page and remove file

File: pages/register.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    I.see('Register Account');
+  },
+
+  fillNewUserForm(user) {
+    I.fillField(this.firstNameField, user.firstName);
+    I.fillField(this.lastNameField, user.lastName); 
+    I.fillField(this.emailField, user.email); 
+    I.fillField(this.phoneField, user.phone); 
+    I.fillField(this.passwordField, user.password); 
+    I.fillField(this.passwordConfirmField, user.passwordConfirm);
```
Comment (adudyak) [inline pages/register.js]: Use user.password as input in confirmation field

File: pages/register.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    I.fillField(this.phoneField, user.phone); 
+    I.fillField(this.passwordField, user.password); 
+    I.fillField(this.passwordConfirmField, user.passwordConfirm); 
+  },
+
+  clickSubscribeLabel() {
+    I.click(this.subscribeLabel);
+  },
+
+  clickontinueButton() {
```
Comment (adudyak) [inline pages/register.js]: Check spelling, looks like a duplicate method

File: tests/register_test.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+const created_account = require("../pages/created_account");
+const { verifyRegisterPageName } = require("../pages/register");
+const register = require("../pages/register");
+
```
Comment (adudyak) [inline tests/register_test.js]: Remove rows 1-3

File: tests/register_test.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+Feature('register');
+
+const NEW_USER = {
+    firstName: "Julia",
+    lastName: "1",
+    email: Date.now() + '@test.com',
+    phone: "+380932849875",
+    password: "eM8x3xTJ4vzVN",
+    passwordConfirm: "eM8x3xTJ4vzVN",
```
Comment (adudyak) [inline tests/register_test.js]: Remove this option for confirm password

File: tests/register_test.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+Scenario('test something',  ({ I, homePage, registerPage, created_accountPage }) => {
+    I.amOnPage('http://opencart.qatestlab.net/index.php');
+    homePage.clickMyAccountButton();
+    homePage.clickMyRegisterLink();
+    registerPage.verifyRegisterPageName();
+    registerPage.fillNewUserForm(NEW_USER);
+    registerPage.clickSubscribeLabel();
+    pause();
```
Comment (adudyak) [inline tests/register_test.js]: Remove pause

**Review by adudyak — APPROVED** [review]

File: pages/register.js (hunk 4 above)
Comment (Julia-92) [reply pages/register.js:33]: Changed to clickPrivacyPolicyRadio

File: pages/created_account.js (hunk 2 above)
Comment (Julia-92) [reply pages/created_account.js]: ++

**Pull request merged**